components/multi-platform-controller/production/kflux-prd-rh02/host-config.yaml

```diff
@@ -392,20 +392,33 @@ data:
 
     --//--
 
-  # Enable After the s390x issue is fixed.
-
-  # dynamic.linux-s390x.type: ibmz
-  # dynamic.linux-s390x.ssh-secret: "ibm-ssh-key"
-  # dynamic.linux-s390x.secret: "ibm-api-key"
-  # dynamic.linux-s390x.vpc: "konflux-prod-multi-rh02"
-  # dynamic.linux-s390x.key: "konflux-s390x-root"
-  # dynamic.linux-s390x.subnet: "sn-20241213-02"
-  # dynamic.linux-s390x.image-id: "r006-20e160c3-58d7-4b3b-827f-9d7994b68095"
-  # dynamic.linux-s390x.region: "us-south-2"
-  # dynamic.linux-s390x.url: "https://us-south.iaas.cloud.ibm.com/v1"
-  # dynamic.linux-s390x.profile: "bz2-2x8"
-  # dynamic.linux-s390x.max-instances: "30"
-  # dynamic.linux-s390x.private-ip: "true"
+# S390x nodes for multi rh02
+
+  dynamic.linux-s390x.type: ibmz
+  dynamic.linux-s390x.ssh-secret: "ibm-ssh-key"
+  dynamic.linux-s390x.secret: "ibm-api-key"
+  dynamic.linux-s390x.vpc: "konflux-prod-multi-rh02"
+  dynamic.linux-s390x.key: "konflux-s390x-root"
+  dynamic.linux-s390x.subnet: "sn-20241213-02"
+  dynamic.linux-s390x.image-id: "r006-20e160c3-58d7-4b3b-827f-9d7994b68095"
+  dynamic.linux-s390x.region: "us-south-2"
+  dynamic.linux-s390x.url: "https://us-south.iaas.cloud.ibm.com/v1"
+  dynamic.linux-s390x.profile: "bz2-2x8"
+  dynamic.linux-s390x.max-instances: "30"
+  dynamic.linux-s390x.private-ip: "true"
+
+  dynamic.linux-s390x.type: ibmz
```

Comment:
Must be `linux-large-s390x` and added into platform list as `linux-large/s390x`

Reply:
Thanks, fixed now